scripts/start_dashboard.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
+
+# Dafault inputs.
+KEEP_ECDSA_SOL_PATH="$PWD/../keep-ecdsa/solidity"
+
+TBTC_SOL_PATH="$PWD/../tbtc/solidity"
+TBTC_SOL_ARTIFACTS_PATH="$TBTC_SOL_PATH/build/contracts"
+
+KEEP_CORE_PATH=$PWD
+KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"
+DASHBOARD_DIR_PATH="$KEEP_CORE_SOL_PATH/dashboard"
+KEEP_CORE_SOL_ARTIFACTS_PATH="$KEEP_CORE_SOL_PATH/build/contracts"
+
+# Run script.
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+cd $KEEP_CORE_SOL_PATH
+
+## uncomment when version of a dependency in package.json has changed.
+# printf "${LOG_START}Installing NPM dependencies...${LOG_END}"
+# rm -f package-lock.json
+# rm -rf node_modules/
+# npm install
+
```

Comment:
Isn't just `npm install` enough here? We are doing it in all other initialization scripts so I'd leave it uncommented.

Reply:
yea, I guess we can leave it uncommented. Changed.